prow/plugins/dco/dco.go

```diff
@@ -97,6 +116,54 @@ type commentPruner interface {
 	PruneComments(shouldPrune func(github.IssueComment) bool)
 }
 
+// checkTrustedUser checks are all commits from a trusted user
+func checkTrustedUser(dco plugins.Dco, gc gitHubClient, l *logrus.Entry, org, repo string, number int) (bool, error) {
```

Comment:
Why are we not using the logic from `trigger`?

Reply:
Fixed. However, I wasn't sure where I should put the `TrustedUser` function. Currently, it's in `prow/plugins/trusteduser.go`, but if there's a better place, let me know. :)